### DISCUSSION TITLE
PDF no se actualiza al querer cargar uno nuevo!

### DISCUSSION THREAD

**Discussion by TomasBruzzesi** [FlowiseAI/Flowise#2236] [Q&A]
Hola, como están? Espero que bien! Estaba teniendo problema cuando en PDF FILE cargo un PDF el cual lo paso como formato texto y se lo envió a ChromaDB, me permite guardarlo y hacerle el Upsert. Cuando quiero hacer el mismo proceso pero como otro PDF me sigue quedando el primer PDF en el Upsert. No me permite cambiarlo, a su vez cada tanto cuando deseo guardarlo, me quita de la pagina directamente y tengo que volver a cagarla.

![Captura desde 2024-04-22 16-56-36](https://github.com/FlowiseAI/Flowise/assets/145077177/7c6fa414-1ce3-473f-8b2a-e0e0a5f78f4c)

![Captura desde 2024-04-22 16-56-07](https://github.com/FlowiseAI/Flowise/assets/145077177/df818e51-0d44-404c-8719-2d7604243473)

![Captura desde 2024-04-22 16-50-37](https://github.com/FlowiseAI/Flowise/assets/145077177/66a89c61-809e-46a5-8aba-bebd8d4ad053)

Criterios de rellamada es el primer PDF que he subido, el segundo es sistemas de información

**Comment by rogerArtefacto** [comment]
no he conseguido replicar el error en mi instalación

si no se guarda igual hay algun problema… mi primera respuesta iba a ser que guardases antes del upsert


elimina el bloque del pdf loader y crea uno nuevo, si sigue sin funcionar duplica el flow igual hay algun problema ahi y por eso no te permite guardarlo
